script/convert_tags_to_topic.rb

```diff
@@ -0,0 +1,57 @@
+require File.expand_path('../../config/environment',  __FILE__)
+require 'colorize'
+
+def put_things_in_tag_to_correct_position cards, skip_year
```

Comment:
Assignment Branch Condition size for put_things_in_tag_to_correct_position is too high. [46.71/15]<br>Method has too many lines. [28/15]